templates/kafka/config.yml

```diff
@@ -0,0 +1,8 @@
+name: Apache Kafka
+description: |
+  (Experimental) Kafka cluster
+version: 0.8.2-rancher1
+category: clustering
+release by: Raul Sanchez
```

Comment:
i don't understand your comments. i made the changes that you request and checks start to fail.  Add '_' where??
Do you have docs with these rules?? it would be easy if it exists.